src/components/board/MemberCard.tsx

```diff
@@ -0,0 +1,11 @@
+import "lucide-react";
+
+const MemberCard = ({ name }: { name: string }) => {
+  return (
+    <div className="flex w-[21vw] flex-col items-center">
+      <p className="mb-[5%] text-[2vw] font-medium">{name}</p>
```

Comment:
```suggestion
      <p className="mb-[5%]">{name}</p>
```
in this case, the text is small and will be pretty responsive either way so just use a twailwind class for the text size